### PR TITLE
[zh]translate learn/starter,supporter,projects,release/10 & update/translate menu

### PR DESCRIPTION
**Pull request by laliwa**
translate learn/starter,supporter,projects,release/10
update/translate menu

### CODE REVIEW

File: content/specifications/_index.md
Comment (oliviergoulet5) [inline content/specifications/_index.md:1]: We need to keep these index pages for /specifications. 

Same for the ZH and JA locales.

File: layouts/shortcodes/pages/learn/starter_guides.html
Comment (oliviergoulet5) [inline layouts/shortcodes/pages/learn/starter_guides.html:1]: This shortcode has been replaced with layouts/shortcodes/guides_list.html. You shouldn't need to re-add the starter_guides.html shortcode.

This was to make the shortcode more generic to be used for other types of guides such as specification guides.

**Review by oliviergoulet5 — CHANGES_REQUESTED** [review]
Aside from some things mentioned below, this PR looks really good!

File: layouts/shortcodes/pages/learn/starter_guides.html
Comment (laliwa) [reply layouts/shortcodes/pages/learn/starter_guides.html:1]: Okay, got that, will notice that in the next commit.

File: content/specifications/_index.md
Comment (laliwa) [reply content/specifications/_index.md:1]: Thanks for that checking, fix missing files now with this commit https://github.com/jakartaee/jakarta.ee/pull/1950/commits/0fa1968a1cff4fe120db9111d8a57b6610159de5

**Review by oliviergoulet5 — APPROVED** [review]

**Comment by oliviergoulet5** [comment]
Thanks @laliwa !

**Pull request merged**